docs/user_guide/assessment.rst

```diff
@@ -258,6 +258,223 @@ across groups and also the difference and ratio between the maximum and minimum:
+across groups. Of the metrics described in this section, it can be the easiest 
+to implement. However, operationalizing fairness using demographic parity 
+rests on a few assumptions: that either the dataset is not a good representation  
+of what the world actually looks like (e.g., a resume assessment system that is 
+more likely to filter out qualified female applicants due to an organizational 
+bias towards male applicants, regardless of skill level), or that the dataset 
+is an accurate representation of the phenomena being modeled, but the 
+phenomena itself is unjust. In reality, this may not be the case. The
+dataset might be an accurate representation of the phenomena itself, 
+or the phenomena being modeled may not be unjust. 
+If either assumption is not true, then demographic parity may not provide 
+a meaningful or useful measurement of the fairness of a model's predictions. 
+
+Fairness metrics like demographic parity can also be used as optimization 
+constraints during the machine learning model training process. However, 
+demographic parity may not be well-suited for this purpose because 
+it does not place requirements on the exact distribution of predictions with 
+respect to other important variables. To understand this concept further, 
+consider an example from the Fairness in Machine Learning textbook :footcite:`fairml_book`. 
+
+    "However, decisions based on a classifier that satisfies independence can 
+    have undesirable properties (and similar arguments apply to other 
+    statistical critiera). Here is one way in which this can happen, 
+    which is easiest to illustrate if we imagine a callous or ill-intentioned 
+    decision maker. Imagine a company that in group a hires diligently 
+    selected applicants at some rate p>0. In group bbb, the company 
+    hires carelessly selected applicants at the same rate p. Even though 
+    the acceptance rates in both groups are identical, it is far more likely 
+    that unqualified applicants are selected in one group than in the other. 
+    As a result, it will appear in hindsight that members of group b 
+    performed worse than members of group a, thus establishing a negative 
+    track record for group b."
+
+It's also worth considering whether the assumptions underlying demographic
+parity maintain construct validity (see :ref:`construct_validity`). 
+Construct validity is a concept in the social sciences that assess the 
+extent to which the ways we choose to measure abstract 
+phenomena is valid. For demographic parity, one relevant question would be 
+whether demographic parity meets the criteria for establishing "fairness", 
+itself an unobservable theoretical construct. Further, it's important 
+to ask whether satisfying demographic parity actually brings us closer 
+to the world we'd like to see. 
+
+
+.. _conditional_group_fairness:
+
+In some cases, we may observe a trend in several groups of data, but that 
```

Comment:
Could we re-word "several groups of data" to "data from multiple demographic groups"? It may make the rest of this a bit clearer (unless that isn't what you mean here)

Reply:
Implemented!